ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-main.yaml

```diff
@@ -19,6 +19,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.23-openshift-4.19
+  yamllint:
+    name: rust
+    namespace: cincinnati-ci-public
+    tag: boot-root-1.79.0
```

Comment:
We need both `make` and `yamlint` and this image happens to have both.
We may switch to a better/smaller one in the future.
For now, the image can do the job.